### PR TITLE
update quickstart scipts

### PR DESCRIPTION
**Pull request by dtthuynh**
Signed-off-by: David Huynh <dhuynh@redhat.com>

# Description of the change(s):
* update quickstart scripts to reflect latest changes

## Why do we need this PR:
*  

## Issue reference: 
* https://issues.redhat.com/browse/ACM-1754

## Test API/Unit - Success
```script

```

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -35,11 +35,16 @@ This document describes how to quickly get started with Hosting Control Planes a
 
 ## What it does
 1. Enables preview_hypershift
-2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to
-3. Imports the `local-cluster`
-4. Applies the Hosting Service Cluster addon (Hypershift) to the `local-cluster` (Hub)
+2. Creates a `local-cluster` `managedCluster` for the OpenShift cluster you are installing to if it doesn't
```
Comment (philipwu08) [inline quickstart/README.md:38]: Something is missing here, either:
1. Creates a `local-cluster` `managedCluster`, if it doesn't already exist
2. Creates a `local-cluster` `managedCluster` to be used as the Hosting Service Cluster, if it doesn't already exist

File: quickstart/README.md [hunk 2]
```diff
@@ -35,11 +35,16 @@ This document describes how to quickly get started with Hosting Control Planes a
...
+3. Imports the `local-cluster` if it doesn't exist
+4. Applies the Hosting Service Cluster addon (Hypershift) to the `local-cluster` (Hub) if it doesn't exist
 
-## Use the hypershift CLI to create a HostedCluster and NodePool
+## Use the oc hcp or hypershift CLI to create a HostedCluster and NodePool
```
Comment (philipwu08) [inline quickstart/README.md:42]: Have we explained what is "OC HCP"  ?

File: quickstart/README.md [hunk 3]
```diff
@@ -35,11 +35,16 @@ This document describes how to quickly get started with Hosting Control Planes a
...
+1. Get the `oc hcp` cli:
```
Comment (philipwu08) [inline quickstart/README.md:43]: Uppercase CLI for consistency

File: quickstart/mce-start.sh [hunk 4]
```diff
@@ -108,9 +109,11 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
-# install hypershift addon
-echo "start to install hypershift addon on the local cluster"
-oc apply -f - <<EOF
+# install hypershift addon if it's not there
+oc get managedclusteraddon hypershift-addon -n local-cluster
+if [ $? -ne 0 ]; then
+  echo "hypershift-addon is not installed on the hub, try to install it"
```
Comment (philipwu08) [inline quickstart/mce-start.sh:115]: "hypershift-addon for local-cluster is not insstalled on the hub, ...:

File: quickstart/acm-start.sh [hunk 5]
```diff
@@ -78,16 +78,19 @@ if [ "${CLUSTER}" == "" ]; then
...
 
-# install hypershift addon
-echo "start to install hypershift addon on the local cluster"
-oc apply -f - <<EOF
+oc create secret generic hypershift-operator-oidc-provider-s3-credentials --from-file=credentials=${S3_CREDS} --from-literal=bucket=${BUCKET_NAME} --from-literal=region=${BUCKET_REGION} -n local-cluster
+
+# install hypershift addon if it's not there
+oc get managedclusteraddon hypershift-addon -n local-cluster
+if [ $? -ne 0 ]; then
+  echo "hypershift-addon is not installed on the hub, try to install it"
```
Comment (philipwu08) [inline quickstart/acm-start.sh:92]: "hypershift-addon for local-cluster is not insstalled on the hub, ...:

File: quickstart/README.md (hunk 2 above)
Comment (philipwu08) [inline quickstart/README.md:42]: Use uppercase for HCP since it's an acronym. 

hypershift - spell HyperShift with the correct cases

**Comment by dtthuynh** [comment]
@philipwu08 @rokej  can you help take a quick look again? ty!

**Pull request merged**